package.json

```diff
@@ -79,25 +80,30 @@
         "style-loader": "^2.0.0",
         "ts-jest": "^26.4.1",
         "ts-loader": "^8.0.5",
-        "typescript": "^4.0.3",
+        "typescript": "^4.4.3",
         "webextension-polyfill-ts": "^0.20.0",
         "webpack": "^4.41.5",
         "webpack-cli": "^3.3.10",
         "webpack-merge": "^4.2.2"
     },
     "dependencies": {
         "@types/classnames": "^2.2.11",
+        "@types/semver": "^7.3.8",
         "bcrypto": "^5.4.0",
         "bdb": "^1.3.0",
         "bignumber.js": "^9.0.1",
         "bootstrap": "^4.5.2",
+        "bsert": "0.0.10",
         "bufio": "^1.0.7",
+        "chrome-net": "^3.3.4",
         "classnames": "^2.2.6",
         "copy-to-clipboard": "^3.3.1",
         "eventemitter2": "^6.4.4",
         "fast-deep-equal": "^3.1.3",
+        "flatted": "^3.2.2",
```

Comment:
Are these 3 dependencies needed? Couldn't find them used in the code anywhere:
```md
- bsert
- chrome-net
- flatted
```

Reply:
Thank you for pointing out! Overlooked in cleanup, will address.